engine/src/main/java/org/terasology/utilities/time/DateTimeHelper.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.utilities.time;
+
+import java.util.concurrent.TimeUnit;
+
+public class DateTimeHelper
```

Comment:
Huh, there isn't some existing utility class available somewhere already? Not necessarily in our code base, but in our dependencies.

If not then maybe a quick tiny bit of class level javadoc, although it is good to see method level javadoc. Seeing two-space indentation here? Copy paste from somewhere?

Reply:
refactored stackoverflow:)
honestly, I didn't find good solution and decided to write my own